ui/markdown_test.go

```diff
@@ -0,0 +1,159 @@
+package ui
+
+import (
+	"testing"
+	"time"
+
+	"github.com/charmbracelet/charm"
+)
+
+func Test_markdownsByLocalFirst_Less(t *testing.T) {
+	a := charm.Markdown{
+		ID:           0,
+		EncryptKeyID: "blabla",
+		Note:         "Blabla",
+		Body:         "blabla",
+		CreatedAt:    time.Now().Local().Add(time.Hour * time.Duration(10)),
+	}
+
+	b := charm.Markdown{
+		ID:           1,
+		EncryptKeyID: "blabla2",
+		Note:         "Blabla2",
+		Body:         "blabla2",
+		CreatedAt:    time.Now().Local(),
+	}
+
+	t.Run("Should returns true if the first markdown is local and the second is not local", func(f *testing.T) {
+		var markdowns markdownsByLocalFirst
+
+		markdowns = append(markdowns, &markdown{
```

Comment:
It may be worth using a table-driven approach here, which could slim down the code by quite a bit.